monitoring.sh

```diff
@@ -5,8 +5,7 @@ requires:
   - boost
   - "GCC-Toolchain:(?!osx)"
   - "ApMon-CPP:(?!osx)"
-  - "system-curl:(slc8)"
```

Comment:
I think monitoring wants  to use system-curl on slc8. Why changing this?

Reply:
@awegrzyn to confirm

Reply:
On slc8, the system curl fails when our OpenSSL is in `LD_LIBRARY_PATH`. So either we install a wrapper script for curl there, or we compile this with our own curl.